src/EprPrnIntegration.Common/Service/EmailService.cs

```diff
@@ -77,14 +77,15 @@ public void SendErrorEmailToNpwd(string errorMessage)
 
     public void SendValidationErrorPrnEmail(string csvData, DateTime reportDate)
     {
+        ArgumentException.ThrowIfNullOrWhiteSpace(csvData, nameof(csvData));
+
         var templateId = _messagingConfig.NpwdValidationErrorsTemplateId;
-        var filename = $"error_events{DateTime.UtcNow.ToShortDateString()}.csv";
         var emailAddress = _messagingConfig.NpwdEmail;
 
         var parameters = new Dictionary<string, object>
         {
-            { "reportDate", reportDate! },
-            { "link_to_file", NotificationClient.PrepareUpload(System.Text.Encoding.UTF8.GetBytes(csvData), filename) }
+            ["reportDate"] = reportDate.ToString("dd/MM/yyyy"), // Ensured UK date format
+            ["link_to_file"] = csvData
```

Comment:
Since now we are doing changes on Template as well, Can we change link_to_file to something like csvData inline with what we are doing